tendrl/commons/objects/cluster/atoms/check_cluster_available/__init__.py

```diff
@@ -41,6 +40,4 @@ def run(self):
                     flow_id=self.parameters['flow_id'],
                     integration_id=NS.tendrl_context.integration_id
                 )
-                raise AtomExecutionFailedError(
-                    "Cluster data sync still incomplete. Timing out"
-                )
+                return False
```

Comment:
log the proper message before return False, just return False again leads to confusion. take the message which is present in AtomExecutionFailedError in old code and log it using logger before return False.

Reply:
It is present, It is present in file, Please try expanding the file

Reply:
looks good